lib/taskana-core/src/test/java/acceptance/priorityservice/TestPriorityServiceProvider.java

```diff
@@ -0,0 +1,27 @@
+package acceptance.priorityservice;
+
+import java.sql.Date;
+import java.time.Instant;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+import pro.taskana.spi.priority.api.PriorityServiceProvider;
+import pro.taskana.task.api.TaskCustomField;
+import pro.taskana.task.api.models.TaskSummary;
+
+public class TestPriorityServiceProvider implements PriorityServiceProvider {
```

Comment:
Do we really need such a complicated TestPriorityServiceProvider?
Why not just return a hard coded priority when custom6 is true? 
This specific implementation makes the test a little harder to understand and the SPI implementation is no concern for us.

Reply:
@holgerhagen @gitgoodjhe